### PR TITLE
Feat/publish post

### PR DESCRIPTION
**Pull request by tissyane**
- Created Page to render User Posts;
- Bearer Token is now provided to publish a new post;
- Profile picture on publish box is provided by LoginContext;
- Translated messages to EN at CreatePost;

### CODE REVIEW

File: src/components/CreatePost.js [hunk 1]
```diff
@@ -35,12 +37,12 @@ export default function CreatePost({ refresh, setRefresh }) {
   function submitData(e) {
     e.preventDefault();
     setDisabled(true);
-    if (form.url === "") {
-      alert("Insira o link que deseja compartilhar");
+    if (form.url === "Please enter a valid URL") {
+      alert("");
```
Comment (henriquesdias) [inline src/components/CreatePost.js]: Why is the alert with an empty string?

Comment (guilhermeicarofr) [reply src/components/CreatePost.js]: I believe she meant:

if ( form.url === "" ) {
      alert("Please enter a valid URL");
}

Comment (tissyane) [reply src/components/CreatePost.js]: Oooop, sorry. You got it, Guilherme. Already fixed it.

File: src/components/App.js [hunk 2]
```diff
@@ -16,25 +17,27 @@ export default function App() {
   );
   const [config, setConfig] = useState(
     localStorage.getItem("linkr")
-      ? { headers: { Authorization: `Bearer ${userData.token}`}}
+      ? { headers: { Authorization: `Bearer ${userData.token}` } }
       : null
```
Comment (SauloDriven) [inline src/components/App.js:21]: **Suggestion:** this logic can be transformed and isolated in a helper, so the app file doesn't get dirty

File: src/components/pages/UserTimeline.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      .then((res) => {
+        setPosts(res.data.posts);
+      })
+      .catch((error) => {
+        console.log(error);
+        if (error.response.status === 404) {
+          setMessage("Nenhum post");
+        }
+      });
+  }, [id]);
```
Comment (SauloDriven) [inline src/components/pages/UserTimeline.js:25]: Nice

**Review by SauloDriven — APPROVED** [review]
:100:

**Review by guilhermeicarofr — APPROVED** [review]
LGTM!

File: src/components/App.js (hunk 2 above)
Comment (tissyane) [reply src/components/App.js:21]: Perfect, we'll do it today

**Pull request merged**